packages/overlay/test/benchmark/directive-test.ts

```diff
@@ -19,36 +19,37 @@ import '@spectrum-web-components/tooltip/sp-tooltip.js';
 import { html, TemplateResult } from 'lit';
 import { measureFixtureCreation } from '../../../../test/benchmark/helpers.js';
 
-const popover = (): TemplateResult => html`
-    <sp-popover>
-        <sp-dialog no-divider>
-            <sp-slider
-                value="5"
-                step="0.5"
-                min="0"
-                max="20"
-                label="Awesomeness"
-            ></sp-slider>
-            <div id="styled-div">
-                The background of this div should be blue
-            </div>
-            <sp-button>
-                Press Me
-                <sp-tooltip
-                    self-managed
-                    delayed
-                >
-                    Click to open another popover.
-                </sp-tooltip>
-            </sp-button>
-        </sp-dialog>
-    </sp-popover>
-`;
+const renderPopover = (): TemplateResult => {
+    import('@spectrum-web-components/popover/sp-popover.js');
+    import('@spectrum-web-components/dialog/sp-dialog.js');
+    import('@spectrum-web-components/slider/sp-slider.js');
+    import('@spectrum-web-components/tooltip/sp-tooltip.js');
```

Comment:
Feels weird again not to `await` any dynamic imports. I think it makes sense that it's on the consumer to deal with their own async content and fallback, but it's not 100% clear to be me why these don't have to be `await`ed.